scripts/scr_bomb_world/scr_bomb_world.gml

```diff
@@ -78,10 +91,11 @@ function scr_bomb_world(star_system, planet_number, bombard_target_faction, bomb
 	    var i=reduced_bombard_score;
 	    roll=0;
     	
-	    if (bombard_protection==0){i=i*4;}
-	    else if (bombard_protection==1){i=i*0.9;}
-	    else if (bombard_protection==2){i=i*0.75;}
-	    else if (bombard_protection==3){i=i*0.5;}
+	    if (bombard_protection==0){i=i*4;} // No protection, Nids out in the open use this
+	    else if (bombard_protection==1){i=i*0.9;} // Poor protection, PDF/Renegades, Sisters and Orks use it,
+	    else if (bombard_protection==2){i=i*0.75;} // Competent protection - IG and standard chaos forces
+	    else if (bombard_protection==3){i=i*0.5;} // Hi-tech, Admech and Tau, And Daemons kind of
+	    else if (bombard_protection==4){i=i*0.34;} // Figured I add a level 4 to this, Ultra hi-tech, Necrons and Eldar
```

Comment:
No, I suppose I could test these changes via debug mode - spawn size 4 orks on a world, bomb with a starting custom fleet, then spawn size 4 Necrons and bomb with the same fleet and figure out the effect that way.

Should I proceed with bombardment tests, and post the results, somewhere?

Reply:
Initially I thought for 4th level to be lowering it to 25%, but figured slightly above a 1/3 would be good enough, at least for now.

If you prefer, I can delete the level 4 and leave the 3 original, and try to set it via that, but I found it difficult to set the proper protection for different factions.

Reply:
As for the logic, it can be seen as this:
0 - cases where the target does not use protection of any kind and just stays out in the open.
1 - A rag-tag group of units, militia-level paramilitaries, that have only rudimentary capabilities and technology.
2 - Standard for functional military. Deep trenches and the like, similar to the great war, perhaps a bit more.
3 - Professionals and folks who have access to more exotic technologies like shield generators.
4 - High-end technologies or psychic capabilities, that make bombardments inefficient.